<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEditDialog</class>
 <widget class="QDialog" name="UserEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменить пользователя</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Сброс пароля</string>
   </property>
   <widget class="QLineEdit" name="PasswordInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordConfirmInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый пароль (еще раз)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PasswordButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить пароль</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Изменение группы</string>
   </property>
   <widget class="QComboBox" name="GroupDropdown">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Администратор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Диспетчер</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Поставщик</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="GroupButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить группу</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новая группа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая группа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CurrentUserGroupInput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Администратор</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Изменение имени пользователя</string>
   </property>
   <widget class="QLineEdit" name="UsernameInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="UsernameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить имя пользователя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новое имя пользователя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CurrentUsernameInput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
